projects/backend/controllers/render.ts

```diff
@@ -240,6 +246,24 @@ const fetchPublishedFront = async (
     return front
 }
 
+const mapSwatchToTheme = (swatch: Swatch) => {
+    switch (swatch) {
+        case 'neutral':
+        case 'news':
+            return Theme.News
+        case 'opinion':
+            return Theme.Opinion
+        case 'culture':
+            return Theme.Culture
+        case 'lifestyle':
+            return Theme.Lifestyle
+        case 'sport':
+            return Theme.Sport
+        default:
+            return null
```

Comment:
@abeddow91 @joecowton1 
sorry, bit late on this, what happens when we pass `null` theme to AR. Shall we return a default `news` theme instead?

Reply:
Also, I presume `Top Special 1` and `Top Special 2` are always either `neutral` or `news`, right?

Reply:
> Also, I presume `Top Special 1` and `Top Special 2` are always either `neutral` or `news`, right?

Always neutral.

Reply:
> @abeddow91 @joecowton1
> sorry, bit late on this, what happens when we pass `null` theme to AR. Shall we return a default `news` theme instead?

We don't want to override an article theme in AR if we don't know which front container it's in. By passing `null` we know that AR will just return the original article style.
AR parser: https://github.com/guardian/apps-rendering/blob/f75b36753290144c38d69296d516f8a313a5d055/src/server/server.ts#L63

Reply:
Great, thanks